.github/workflows/merge-master-into-develop.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         run: git checkout develop
       - name: Check for merge conflict
         id: check-conflict
-        run: echo "::set-output name=merge_conflict::$(git merge-tree $(git merge-base HEAD master) master HEAD | egrep '<<')"
+        run: echo "::set-output name=merge_conflict::$(git merge-tree $(git merge-base HEAD master) master HEAD | egrep '<<<<<<<')"
```

Comment:
I aim surprised there is no GitHub API to look at the files flagged as conflict in git.